packages/web/src/common/store/backend/sagas.ts

```diff
@@ -78,7 +78,7 @@ export function* setupBackend() {
   const audiusBackendInstance = yield* getContext('audiusBackendInstance')
 
   // Init APICLient
-  yield* call(() => apiClient.init())
+  apiClient.init()
```

Comment:
Why remove the yield here when we made `init` async?

Reply:
oh yeah i can bring that back, i actually didnt make it async since it was messing up a lot of places and for very little upside, which is just to init the eager discovery provider, which is only a web thing anyway.